libstorage/RocksDBStorage.cpp

```diff
@@ -148,9 +161,10 @@ void RocksDBStorage::setDB(std::shared_ptr<rocksdb::DB> db)
     m_db = db;
 }
 
-void RocksDBStorage::processEntries(h256 hash, int64_t num,
-    std::shared_ptr<std::map<std::string, Json::Value> > key2value, TableInfo::Ptr tableInfo,
-    Entries::Ptr entries)
+void RocksDBStorage::processNewEntries(h256 hash, int64_t num,
+    std::shared_ptr<std::map<std::string, std::vector<std::map<std::string, std::string>>>>
```

Comment:
recommend `using xxx = std::map<std::string, std::vector<std::map<std::string, std::string>>>`, the same as above.

Reply:
ok